doc/changes/changes49.xml

```diff
@@ -0,0 +1,1196 @@
+<Chapter Label="ChangesGAP48toGAP49"> 
+<Heading>Changes between &GAP; 4.8 and &GAP; 4.9</Heading>
+
+This chapter contains an overview of the most important changes
+introduced in &GAP; 4.9.1 release (the 1st public release of &GAP; 4.9).
+Later it will also contain information about subsequent update 
+releases for &GAP; 4.9.
+<P/>
+
+These changes are also listed on the Wiki page
+<URL>https://github.com/gap-system/GAP/wiki/gap-4.9-release-notes</URL>.
+
+<Section Label="GAP491">
+<Heading>&GAP; 4.9.1 (May 2018)</Heading>
+
+<Subsection Label="Changes in the core GAP49 system"> 
+<Heading>Changes in the core &GAP; system introduced in &GAP; 4.9</Heading>
+
+Major changes:
+
+<List>
+<Item>
+Merged &HPCGAP; into &GAP;. For details, please refer to the &HPCGAP; section at
+the end of these release notes.
+</Item>
+<Item>
+&GAP; has a new build system, which resolves many quirks and issues with the
+old system, and will be easier to maintain. For regular users, the usual 
+<C>./configure &amp;&amp; make</C> should work fine as before.
+If you are interested in technical details on the new build system,
+take a look at <File>README.buildsys.md</File>.
+</Item>
+<Item>
+The guidelines for developing &GAP; packages were revised and moved from the
+Example package to <Ref BookName="ref" Label="Using and Developing GAP Packages"/>
+(<URL><LinkText>&Hash;484</LinkText><Link>https://github.com/gap-system/gap/pull/484</Link></URL>).
+</Item>
+<Item>
+In addition to supporting single argument lambda functions like 
+<C>a -> a+1</C>, &GAP; now supports lambdas with fewer or more than
+one argument, or even an arbitrary number. E.g. <C>{a,b} -> a+b</C>
```

Comment:
I think "an arbitrary number" should be changed to "a variable number" (also in the Wiki) for clarity